pkg/kubectl-argo-rollouts/info/info.go

```diff
@@ -55,13 +57,20 @@ func ownerRef(ownerRefs []metav1.OwnerReference, uids []types.UID) *metav1.Owner
 	return nil
 }
 
-func parseRevision(annotations_ map[string]string) int {
+func parseRevision(annotations_ map[string]string) int32 {
```

Comment:
why do we need to change this function?

Reply:
This all are related to the parseRevision function, which was not able to handle the boundary check. int was not able to handle the limits, so needed to change to int32. and change the function to handle boundary checks.